src/chai-form.ts

```diff
@@ -312,15 +344,37 @@ export class ChaiForm extends LitElement {
     `;
   }
 
+  private lastFormLoadIsRecent() {
+    return (
+      localStorage.getItem('chai-form-load') != null &&
```

Comment:
Don't call `getItem` twice. Get it once and evaluate it against both conditions.